.github/workflows/maintenance-bundle-update.yml

```diff
@@ -0,0 +1,37 @@
+name: Maintenance - Bundle Update
+
+on:
+  workflow_dispatch
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  bundle-update:
+    runs-on: windows-latest
+    env:
+      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.2 # Use same Ruby version with fluent-package bundled
+      - uses: actions/checkout@v4
+      - name: Bundle Update
+        run: |
+          cd fluent-package
+          gem install bundler:2.3.27
+          bundle _2.3.27_ update -j 4
+      - name: Create Pull Request
+        run: |
+          $today=Get-Date -Format "yyyy-MM-dd"
+          $branch="maintenance/bundle-update/${today}"
+
+          git config user.name github-actions
+          git config user.email github-actions@github.com
```

Comment:
I've heard that github-actions@github.com is not recognized appropriate.

https://github.com/orgs/community/discussions/26560

Reply:
ref. 

https://github.com/actions/checkout?tab=readme-ov-file#push-a-commit-to-a-pr-using-the-built-in-token

README: Suggest user.email to be 41898282+github-actions[bot]@users.noreply.github.com
https://github.com/actions/checkout/pull/1707

Reply:
Thanks!